packages/expo-notifications/android/src/main/java/expo/modules/notifications/badge/BadgeModule.kt

```diff
@@ -0,0 +1,37 @@
+package expo.modules.notifications.badge
+
+import android.content.Context
+import android.util.Log
+import expo.modules.core.ExportedModule
+import expo.modules.core.Promise
+import expo.modules.core.interfaces.ExpoMethod
+import me.leolin.shortcutbadger.ShortcutBadgeException
+import me.leolin.shortcutbadger.ShortcutBadger
+
+class BadgeModule(context: Context) : ExportedModule(context) {
+  override fun getName(): String = "ExpoBadgeModule"
+
+  @ExpoMethod
```

Comment:
Why not to rewrite this small module already? 😉

Reply:
I split the changes across multiple PRs, and the package was written in Java. So, it will be easier to convert exported modules to Kotlin first to check if that translation won't introduce any regressions. Later, I will introduce a new API.